package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Project that contains all hosted contents of Dynamo Windows client",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "set NODE_ENV=0 && webpack --progress --colors",
+    "build": "set NODE_ENV=1 && webpack"
```

Comment:
Remember to rename `build` to `prod`. Also rename `NODE_ENV` to `production_build`, aligning better with the rest (also *node env* doesn't really mean anything).